python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+else:
+    path=''
+
+f=glob.glob(path+"/meta/*ch_location*.txt")
```

Comment:
use the `os.path` tools, and `os.path.join` to create and maintain paths.

Reply:
Added os.path for joining these paths. I assume you have no issue with the use of glob here